Source/PeleLMeX_K.H

```diff
@@ -440,6 +440,31 @@ getMwmixGivenRY(
   Mwmix(i, j, k) = Mwmix(i, j, k) * 0.001_rt; // CGS -> MKS conversion
 }
 
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+getGradMwmixGivengradYMwmix(
+  int i,
+  int j,
+  int k,
+  amrex::Array4<const amrex::Real> const& gradY,
+  amrex::Array4<const amrex::Real> const& Mwmix,
+  amrex::Array4<amrex::Real> const& gradMwmix) noexcept
+{
+  using namespace amrex::literals;
+
+  auto eos = pele::physics::PhysicsType::eos();
+  amrex::Real imw[NUM_SPECIES] = {0.0_rt};
+  eos.inv_molecular_weight(imw);
+  gradMwmix(i, j, k) = 0.0;
+  for (int n = 0; n < NUM_SPECIES; n++) {
+    imw[n] *= 1000.0; // CGS -> MKS
```

Comment:
I thought this was correct, but if I comment out, everything converges nice and quickly - thoughts?

Reply:
This reflects me being away from things for a while, but the MKS version of mw *could* be kg/kmole (depending on how it is used elsewhere) so that there is no conversion factor necessary.  :)

Reply:
That is weird because I do think multiplying by 1000 is correct here. When computing MWmix we divide by 1000. Perhaps there's some error cancellation going on?